### PR TITLE
Make white headers possible

### PR DESCRIPTION
**Pull request by wielinde**
https://community.openproject.com/projects/gmbh/work_packages/25275/activity

### CODE REVIEW

**Comment by oliverguenther** [comment]
Killed the PR run to make room for a high-prio bug. Please restart if you need the test results for this commit

File: app/views/custom_styles/show.html.erb [hunk 1]
```diff
@@ -77,17 +77,32 @@ See doc/COPYRIGHT.rdoc for more details.
...
+                             class: 'timelines-x-update-color timelines-color-properties-preview',
+                             data:  { target: '#color_hexcode' } %>
           <div class="form--field-instructions">
             <% if design_color.variable == "primary-color" %>
               <%= t("admin.custom_styles.instructions.primary_color") %>
             <% elsif design_color.variable == "primary-color-dark" %>
               <%= t("admin.custom_styles.instructions.primary_color_dark") %>
             <% elsif design_color.variable == "alternative-color" %>
               <%= t("admin.custom_styles.instructions.alternative_color") %>
+            <% elsif design_color.variable == "header-bg-color" %>
```
Comment (oliverguenther) [inline app/views/custom_styles/show.html.erb:96]: This is starting to get out of hand. What's wrong with `design_color.variable.tr('-', '_')` ? :-)

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/1052bce7e129b1c3ade38910ed5dc65937378d16) (deployed commit: 8259b9907442910a1fabd5f11ae76018ec0c5b4e).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/4628481526b507f7da864d21a480c074862fe34c) (deployed commit: 8259b9907442910a1fabd5f11ae76018ec0c5b4e).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/091da1ea43d745071616b7a531451cae4ef49494) (deployed commit: e8389b8f3dfbf65057fbf3e28967b4d90fe8a533).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/e4df1df626c80e5830989080d5d20ce68bb67673) (deployed commit: 73fe441e71e51ae0acc4f17fe01b4a1f60a2ed92).

**Comment by crohr** [comment]
A new preview environment is ready! [Click to open](https://pullpreview.com/opf/openproject/sessions/15b9290eaf51f17e59449fa31f6842ebe6e727d3) (deployed commit: d2def0ff7f8aae371e8cd3199655b23ace78e261).

**Pull request merged**